SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -660,6 +661,22 @@ ERPCResult USpatialSender::SendRPCInternal(UObject* TargetObject, UFunction* Fun
 
 		Worker_CommandRequest CommandRequest = CreateRPCCommandRequest(TargetObject, Payload, ComponentId, RPCInfo.Index, EntityId);
 
+		// This check happens after building the command request as building the request also sets the entity ID.
+		if (ComponentId == SpatialConstants::SERVER_TO_SERVER_ENDPOINT_COMPONENT_ID_LEGACY)
```

Comment:
We were not doing these checks before, and this meant that rpcs were quite happily being sent to workers that weren't listening. Not sure if this was a problem or not, and if there was some reason why the checks weren't being done, please shout. This appears to work as expected, however.

Reply:
dropped the check